Headers/AppKit/NSView.h

```diff
@@ -123,6 +124,9 @@ typedef enum _NSFocusRingType {
   NSFocusRingTypeExterior = 2
 } NSFocusRingType;
 
+extern const CGFloat NSViewNoInstrinsicMetric;
```

Comment:
Why do we need this misspelled version? And would a `#define` be enough here?

Reply:
It was baked into the original header file and fixed in a new version. So we need to support both versions.